lib/plugins/inspec-habitat/README.md

```diff
@@ -0,0 +1,150 @@
+# InSpec Habitat Plugin
+
+## Summary
+
+This plugin allows you to do the following:
+  1. Add Habitat configuration to a profile
+  2. Create/Upload a Habitat package from an InSpec profile
+
+Creating a [Habitat](https://www.habitat.sh/) package from an InSpec profile
+allows you to execute that profile as a service (via a Habitat Supervisor) on
+any Linux based platform.
+
+When running as a service, an InSpec report will be created in JSON format (by
+default at `/hab/svc/YOUR_SERVICE/logs/inspec_last_run.json`).  Additionally, a
+log of the last run will be located at
+`/hab/svc/YOUR_SERVICE/logs/inspec_log.txt` and CLI output is viewable in
+the Supervisor logs by default. You can also configure this service to report
+to [Chef Automate](https://www.chef.io/automate/).
+
+See below for usage instructions.
+
+## Plugin Usage
+
+### Adding Habitat Configuration to an InSpec Profile
+
+Run the following command:
+
+```
+inspec habitat profile setup PATH
```

Comment:
When I run this, I see no output at all.  Is that intentional?

Reply:
Hmm...I think it was. https://github.com/inspec/inspec/pull/3818/commits/27eab8686ffda6645ff2a65e23e661abe5ac1184

I see your point though. I'll change it to output at least 1 message.